buildutils/attr_parser.py

```diff
@@ -90,762 +94,191 @@ def match(self, *args):
             return False
 
 
-def fileappend(line):
+def fileappend(prop_type, line):
     """
     fileappend function - (wrapper on top of append for being able to
     select the file where to write
     """
-    global s_flag
-    global e_flag
-
-    if s_flag == 1 and e_flag == 0:
-        list_svr.append(line)
-    if e_flag == 1 and s_flag == 0:
-        list_ecl.append(line)
-    if e_flag == 0 and s_flag == 0:
-        list_svr.append(line)
-        list_ecl.append(line)
+    global attr_type
+
+    if prop_type == PropType.SERVER:
+        if attr_type == PropType.SERVER or attr_type == PropType.BOTH:
+            list_svr.append(line)
+    elif prop_type == PropType.ECL:
+        if attr_type == PropType.ECL or attr_type == PropType.BOTH:
+            list_ecl.append(line)
+    elif prop_type == PropType.BOTH:
+        if attr_type == PropType.SERVER or attr_type == PropType.BOTH:
+            list_svr.append(line)
+        if attr_type == PropType.ECL or attr_type == PropType.BOTH:
+            list_ecl.append(line)
     return None
 
 
-def getText(efl, sfl):
+def getText(s_file, e_file, d_file):
```

Comment:
how about server_file, ecl_file, def_file?

Reply:
just variables, see the caller - should explain everything